src/test/java/io/ipfs/api/APITest.java

```diff
@@ -64,6 +69,14 @@ public void keys() throws IOException {
         Assert.assertTrue("removed key", remaining.equals(existing));
     }
 
+    @Test
+    public void log() throws IOException {
+        Map lsResult = ipfs.log.ls();
+        Assert.assertTrue("Log ls", !lsResult.isEmpty());
+        Map levelResult = ipfs.log.level("all", "info");
+        Assert.assertTrue("Log level", ((String)levelResult.get("Message")).startsWith("Changed log level"));
```

Comment:
Is this reliable?

Reply:
have resolved the failing test issue in latest